### PR TITLE
Importing data files

### PR DESCRIPTION
**Pull request by sowmyaraj92**
Updated screenshots

### CODE REVIEW

File: src/pages/docs/running-collections/working-with-data-files.md [hunk 1]
```diff
@@ -30,32 +17,32 @@ You can use data files to pass Postman sets of values to use in a collection run
 
 ## Running collections with data files
 
-When you [initiate a collection run](/docs/running-collections/intro-to-collection-runs/) you will see an option to select a data file. Click __Runner__ at the top left of Postman. Select your collection, and click __Select File__ next to __Data__.
+When you [initiate a collection run](/docs/running-collections/intro-to-collection-runs/) you will see an option to select a data file. Click __Runner__ at bottom right of Postman. Select your collection, and click __Select File__ next to __Data__.
 
-![Setup Collection Run](https://assets.postman.com/postman-docs/data-file-run-setup.png)
+![Collection Runner](https://assets.postman.com/postman-docs/collection-runner-v8.gif)
```
Comment (claireriley) [inline src/pages/docs/running-collections/working-with-data-files.md:22]: I think the start of this gif has a few blank frames which is a bit confusing, could we cut those?

Comment (claireriley) [reply src/pages/docs/running-collections/working-with-data-files.md:22]: Actually also could we include you clicking **Select File** in this?

File: src/pages/docs/running-collections/working-with-data-files.md [hunk 2]
```diff
@@ -30,32 +17,32 @@ You can use data files to pass Postman sets of values to use in a collection run
 
 ## Running collections with data files
 
-When you [initiate a collection run](/docs/running-collections/intro-to-collection-runs/) you will see an option to select a data file. Click __Runner__ at the top left of Postman. Select your collection, and click __Select File__ next to __Data__.
+When you [initiate a collection run](/docs/running-collections/intro-to-collection-runs/) you will see an option to select a data file. Click __Runner__ at bottom right of Postman. Select your collection, and click __Select File__ next to __Data__.
```
Comment (claireriley) [inline src/pages/docs/running-collections/working-with-data-files.md:20]: It took me a bit to understand where **Select File** was. Could we adjust the second sentence to be a bit more explicit? Like

"Select your collection, then click __Select File__ next to __Data__ on the right."

File: src/pages/docs/running-collections/working-with-data-files.md [hunk 3]
```diff
@@ -30,32 +17,32 @@ You can use data files to pass Postman sets of values to use in a collection run
...
-Click a request in the Collection Runner __Run Results__ to see more detail on its data.
+Click a request in the Collection Runner results to see more detail on its data.
 
-![Collection Run Results](https://assets.postman.com/postman-docs/data-file-collection-run.png)
+![Collection Run Results](https://assets.postman.com/postman-docs/data-file-collection-run-v8.jpg)
 
 Any data you have defined in the requests will be used when the collection runs, and your request data can reference values defined in the data file.
 
-![Data File Value](https://assets.postman.com/postman-docs/request-body-data-run.png)
+![Data File Value](https://assets.postman.com/postman-docs/request-body-data-run-v8.jpg)
```
Comment (claireriley) [inline src/pages/docs/running-collections/working-with-data-files.md:45]: This last screenshot is a bit random to me, might consider removing

**Review by claireriley — APPROVED** [review]
Looks great @sowmyaraj92 ! Few comments to check out but no blockers so good to merge once you do

File: src/pages/docs/running-collections/working-with-data-files.md (hunk 1 above)
Comment (sowmyaraj92) [reply src/pages/docs/running-collections/working-with-data-files.md:22]: Cropped the blank frames. Added an extra screenshot for **Select File**.

File: src/pages/docs/running-collections/working-with-data-files.md (hunk 3 above)
Comment (sowmyaraj92) [reply src/pages/docs/running-collections/working-with-data-files.md:45]: It is to indicate request data can reference values.

**Pull request merged**